panel_view.py

```diff
@@ -72,6 +72,27 @@ def on_pre_close(self, view):
         if window:
             sublime.set_timeout_async(lambda: fill_panel(window, update=True))
 
+    def on_post_save_async(self, view):
+        if persist.settings.get('show_panel_on_save') == 'never':
+            return
+
+        window = view.window()
+        panel = ensure_panel(window)
+        # If we're here and the user actually closed the window in the meantime,
+        # we cannot create a panel anymore, and just pass.
+        if not panel:
+            return
+
+        errors_by_bid = get_window_errors(window, persist.errors)
+
+        if persist.settings.get('show_panel_on_save') == 'window' and errors_by_bid:
+            window.run_command('show_panel', {'panel': 'output.' + PANEL_NAME})
+        else:
+            for bid in errors_by_bid:
+                if bid is view.buffer_id():
+                    window.run_command('show_panel', {'panel': 'output.' + PANEL_NAME})
```

Comment:
Now, iirc for `errors_by_bid` is a dict so you can basically test `view.buffer_id() in errors_by_dict`.

But note that as soon as we have a lazy panel_view it is not enough to just show it. So firing our own command `sublime_linter_panel_toggle` with `force_show=True` is probably the right thing to do.

Reply:
Calling our own command is not a bad idea anyway, so let's do that.